src/main/resources/application.yml

```
@@ -17,11 +20,22 @@ spring:
       exposure:
         include: "*"
+      domain: dds
     web:
       exposure:
         include: "health,info,metrics"
-
+debezium:
+  offset-storage-dir: /tmp/debezium/offsets
+  offset-storage-file-name: debezium-offsets.dat
+  history-dir: /tmp/debezium/history
+  history-file-name: debezium-history.dat
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)